how-to/customize-workspace/client/src/modules/composite/windows/shapes.ts

```diff
@@ -0,0 +1,72 @@
+import type { MenuPosition, MenuSeparatorPosition } from "customize-workspace/shapes";
+
+export interface WindowMenuSettings {
+	/** Settings related to the show window menu entry */
+	showAllWindows?: {
```

Comment:
Could we have a new type which include the label, icon, include and position, instead of all the duplication

Reply:
I added a new type at the module level and did the same for pages as well.